com.github.theCalcaholic.StreamDeck.yml

```diff
@@ -0,0 +1,82 @@
+app-id: com.github.theCalcaholic.StreamDeck
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/streamdeck.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --filesystem=/run/udev:ro
+  - --filesystem=xdg-data/Steam:rw
+  - --filesystem=~/.steam:rw
+  - --filesystem=~/.mozilla/firefox:rw
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:rw
+  - --filesystem=~/.var/app/org.mozilla.firefox:rw
+  - --filesystem=xdg-data/flatpak:ro
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=STREAMDECK_CMD_PREFIX=flatpak-spawn --host
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+modules:
+  - name: mtdev
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://bitmath.org/code/mtdev/mtdev-1.1.6.tar.gz
+        sha256: 1325f389a2f25cd5f5a8ea4d29aad24aa7c3ec30401d679400dd79eb9c0a8dbb
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} config-aux
+  - name: xclip
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/astrand/xclip.git
+    modules:
+      - name: libx11-dev
+        buildsystem: autotools
+        sources:
+          - type: archive
+            url: http://archive.ubuntu.com/ubuntu/pool/main/libx/libx11/libx11_1.6.9.orig.tar.gz
+            sha256: b8c0930a9b25de15f3d773288cacd5e2f0a4158e194935615c52aeceafd1107b
+      - name: libxmu-headers
+        buildsystem: autotools
+        sources:
+          - type: archive
+            url: http://archive.ubuntu.com/ubuntu/pool/main/libx/libxmu/libxmu_1.1.3.orig.tar.gz
+            sha256: 5bd9d4ed1ceaac9ea023d86bf1c1632cd3b172dce4a193a72a94e1d9df87a62e
+  - name: streamdeck
+    buildsystem: simple
+    sources:
+      - type: dir
+        path: streamdeck
+    build-commands:
+      - pip3 install --prefix=/app --root=/ .
+    modules:
+      - python3-requirements.json
+  - name: StreamDeck Wrapper and appstream data
+    buildsystem: simple
+    sources:
+      - type: dir
+        path: streamdeck/flatpak
+    build-commands:
+      - install -D streamdeck.sh /app/bin/streamdeck.sh
+      - install -Dm644 com.github.theCalcaholic.StreamDeck.appdata.xml -t /app/share/metainfo/
+  - name: StreamDeck shared resources
+    buildsystem: simple
+    sources:
+      - type: dir
+        path: streamdeck/share
+    build-commands:
+      - |
```

Comment:
```suggestion
      - install -D streamdeck.sh /app/bin/streamdeck.sh
      - install -Dm644 com.github.theCalcaholic.StreamDeck.appdata.xml -t /app/share/metainfo/
      - |
```

Reply:
I couldn't use your changes right away, because the files are in different subdirectories. But I have merged the modules into one (and tested the result)